src/cmd/kssh/kssh.go

```diff
@@ -257,20 +280,61 @@ func provisionNewKey(config kssh.ConfigFile, keyPath string) error {
 	return nil
 }
 
-// Run SSH with the given key. Calls os.Exit if SSH returns
+// Run SSH with the given key. Calls os.Exit and does not return.
 func runSSHWithKey(keyPath string, remainingArgs []string) {
-	fmt.Printf("\n") // a new line to separate kssh output from ssh output
+	// Create a config file for the default user if necessary
```

Comment:
oh, this should be down a few lines.

Reply:
Tweaked the comment. It is meant to label that chunk of code as handling the default ssh user configuration.